nexus_integration_tests/launch/control_center.launch.py

```diff
@@ -132,6 +132,7 @@ def launch_setup(context, *args, **kwargs):
                     """{
                         pick_and_place: [place_on_conveyor, pick_from_conveyor],
                     }""",
+                "main_bt_filename": "main.xml",
```

Comment:
Please `DeclareLaunchArgument` and get the value via `LaunchConfiguration` instead of hardcoding here. That way, parent launch files like `launch.py` can overwrite the value if needed.

Reply:
Ah totally forgot, https://github.com/osrf/nexus/pull/47/commits/99c0c6902004dbbc1bae4cd347cd43b93de41069. It's not declared in the root `launch.py` but it is still passed to the `control_center.launch.py`, I can declare it in both places if it makes it clearer